payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/CardDetailsSectionElementUI.kt

```diff
@@ -0,0 +1,62 @@
+package com.stripe.android.ui.core.elements
+
+import androidx.annotation.RestrictTo
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.heading
+import androidx.compose.ui.semantics.semantics
+import com.stripe.android.stripecardscan.cardscan.CardScanSheetResult
+import com.stripe.android.stripecardscan.cardscan.exception.UnknownScanException
+import com.stripe.android.ui.core.DefaultIsStripeCardScanAvailable
+import com.stripe.android.ui.core.R
+import com.stripe.android.ui.core.cardscan.CardScanActivity
+
+@Composable
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+fun CardDetailsSectionElementUI(
+    enabled: Boolean,
+    controller: CardDetailsSectionController,
+    hiddenIdentifiers: List<IdentifierSpec>?
+) {
+    val isStripeCardScanAvailable = DefaultIsStripeCardScanAvailable()
```

Comment:
This should be passed as a parameter, or even just pass the boolean directly. Since this is a composable function, it's called every time the UI needs to be updated, so a new object is instantiated. I think the right place for it is in FormViewModel.